js/components/ManuscriptForm.js

```diff
@@ -203,31 +198,25 @@ class ManuscriptForm extends React.Component {
             {" Untrimmed"}
           </label>
         </div>
-        <div className={"field is-horizontal"}>
-          <div className={"field-label is-normal"}>
-            <div style={{ whiteSpace: "nowrap" }}>Image size:</div>
-          </div>
-          <div className={"field-body"}>
-            <div className={"field is-narrow"}>
-              <div className={"control"}>
-                <div className={"select is-small is-fullwidth"}>
-                  <select
-                    value={this.state.imageSize}
-                    type="string"
-                    name="imageSize"
-                    onChange={this.handleChange}
-                  >
-                    <option>Small</option>
-                    <option>Medium</option>
-                    <option>Large</option>
-                  </select>
-                </div>
-              </div>
-            </div>
+
+        <div className={"field is-horizontal flex-row"}>
+          <div className={"control"}>Image size: </div>
```

Comment:
I wonder whether we should be using better semantic HTML, like <label> tags for some of these form elements. Thoughts?

Reply:
Sure, I'll give it a try -- the current implementation definitely prioritizes functionality over any kind of semantic description, so we at least can address some easy wins like more consistent use of label tags (to be honest I'm not sure what else to do besides adding label tags, but will look around for other opportunities).

Reply:
I'm not sure what else to do beyond using the semantic elements for forms, mainly labels, for the moment either. Let's do that if possible, and then merge it to move forward